lib/ansible/modules/windows/win_package.ps1

```diff
@@ -424,15 +425,20 @@ if ($state -eq "absent") {
             }
 
             if (-not $check_mode) {
-                $install_command = Argv-ToString -arguments $install_arguments
+                $command_args = @{
+                    command = Argv-ToString -arguments $install_arguments
+                }
                 if ($arguments -ne $null) {
-                    $install_command += " $arguments"
+                    $command_args['command'] += " $arguments"
+                }
+                if ($chdir) {
+                    $command_args['working_directory'] = $chdir
                 }
 
                 try {
-                    $process_result = Run-Command -command $install_command
+                    $process_result = Run-Command @command_args
                 } catch {
-                    Fail-Json -obj $result -message "failed to run install process ($install_command): $($_.Exception.Message)"
+                    Fail-Json -obj $result -message "failed to run install process ($command_args['command']): $($_.Exception.Message)"
```

Comment:
Do you need to do `$($command_args['command'])` here?

Reply:
This change is because `$install_command` is now called `$command_args['command']` due to the change on line 433 so this is not a change to output.

Reply:
Yep, I can see why the variable change but I believe you need to enclose the full variable like `$($command_args['command'])` due to how PowerShell interprets vars inside a string.

Reply:
Gotcha. Well spotted. This is resolved in the latest commit.